db/seeds.rb

```diff
@@ -31,6 +31,14 @@
   r.time_zone = 'America/New_York'
 end.save!
 
+Location.find_or_initialize_by(
+  city_name: 'SanDiego'
+) do |r|
+  r.latitude = 32.7157
+  r.longitude = -117.1611
+  r.time_zone = 'America/Los_Angeles'
```

Comment:
What is the '|r|' and why do we need it?

Reply:
So, from reading up on this, it's a form of saving values to each block of code. In this case, we're creating a new Location object for San Diego with multiple attributes. `do |r|`goes through each variable and saves their values to the new object. Mentors, is my statement correct? I may be on the right track but I'm not 100% positive on my response.